.gitignore

```diff
@@ -3,6 +3,7 @@ cmdline.h
 version.c
 docs/html/
 
+.idea
```

Comment:
Typo? Please remove.

Reply:
It's not a typo, CLion editor internally stores project config in .idea directory

Reply:
Then please move it at the end of the file, and add a comment specifying what it's for (see OSX line).